util/cron/test-xc-wb.bash

```diff
@@ -7,15 +7,14 @@ source $CWD/functions.bash
 source $CWD/common-whitebox.bash
 source $CWD/common-localnode-paratest.bash
 
-# Whitebox testing runs multiple paratests concurrently (-nodepara 2 for up to
-# 4 configs.) We limit the number of running executables, but we use a file as
-# a lock to do that, which is racey. Limit the amount of spinwaiting we do and
-# turn off affinity since they'll be a lot compiles and maybe 2 executables
-# running concurrently. Note that we don't disable affinity for pgi because
-# its lack of atomics combined with no affinity has led to timeouts.
-export QT_SPINCOUNT=300
-if [ "${COMPILER}" != "pgi" ] ; then
-  export QT_AFFINITY=no
+source $CWD/common-oversubscribed.bash
```

Comment:
I think I would put this down below, after the block comment rather than before it.  Or at the very least, after the first 3 sentences of the block comment.